eliot/_action.py

```diff
@@ -8,12 +8,12 @@
 from __future__ import unicode_literals, absolute_import
 
 import threading
-from uuid import uuid4
+from uuid import UUID, uuid4
 from itertools import count
 from contextlib import contextmanager
 from warnings import warn
 
-from pyrsistent import PClass, pvector_field
+from pyrsistent import field, PClass, optional, pmap_field, pvector_field, m, v
```

Comment:
There we have a difference of opinion. Changed to use `pmap` and `pvector`, which are equivalent in the case of constructing empty containers.